rspec-tools/rspec_tools/validation/description.py

```diff
@@ -14,13 +16,41 @@
 SECTION_NAMES_PATH = Path(__file__).parent.parent.parent.parent.joinpath('docs/section_names.adoc')
 SECTION_NAMES_FILE = SECTION_NAMES_PATH.read_text(encoding='utf-8').split('\n')
 ACCEPTED_SECTION_NAMES: Final[list[str]] = [s.replace('* ', '').strip() for s in SECTION_NAMES_FILE if s.strip()]
+# The list of all the framework names currently accepted by the script.
+FRAMEWORK_NAMES_PATH = Path(__file__).parent.parent.parent.parent.joinpath('docs/allowed_framework_names.adoc')
+FRAMEWORK_NAMES_FILE = FRAMEWORK_NAMES_PATH.read_text(encoding='utf-8').split('\n')
+ACCEPTED_FRAMEWORK_NAMES: Final[list[str]] = [s.replace('* ', '').strip() for s in FRAMEWORK_NAMES_FILE if s.strip()]
+
 def validate_section_names(rule_language: LanguageSpecificRule):
   descr = rule_language.description
   for h2 in descr.find_all('h2'):
     name = h2.text.strip()
     if name not in ACCEPTED_SECTION_NAMES:
       raise RuleValidationError(f'Rule {rule_language.id} has unconventional header "{name}"')
 
+def validate_how_to_fix_it_subsections(rule_language: LanguageSpecificRule):
+  descr = rule_language.description
+  frameworks_counter = 0
+
+  for h3 in descr.find_all('h3'):
+    name = h3.text.strip()
+    # It is important that the Regex here matches the one used by the analyzers when loading the rules content
+    result = re.search('How to fix it in (?:(?:a|an|the)\\s)?(.*)', name)
```

Comment:
Here it should be irrelevant, but just to be consistent with what will be in the analyzer you can modify like this:
```suggestion
    result = re.search('How to fix it in (?:(?:an|a|the)\\s)?(.*)', name)
```